src/main/rx/xbus.c

```diff
@@ -158,9 +181,68 @@ static void xBusUnpackFrame(void)
 
 }
 
+static void xBusUnpackRJ01Frame(void)
+{
+    // Calculate the CRC of the incoming frame
+    uint16_t crc = 0;
+    uint16_t inCrc = 0;
+    uint8_t i = 0;
+    uint16_t value;
+    uint8_t frameAddr;
+
+    // When using the Align RJ01 receiver with 
+    // a MODE B setting in the radio (XG14 tested)
+    // the MODE_B -frame is packed within some
+    // at the moment unknown bytes before and after:
+    // 0xA1 __ __ 0xA1 12*(High + Low) CRC1 CRC2 + __ __ __
```

Comment:
On my logic analyzer, I get "frames" with data looking like this:
0xA1,0x1E,0x06,0xA1,0x07,0xD6,0x08,0x09,0x08,0x00,0x01,0xB7,0x01,0xB7,0x0E,0x48,0x0E,0x48,0x0E,0x48,0x0E,0x48,0x0E,0x48,0x0E,0x48,0x0E,0x48,0x4C,0xBA,0xC8,0xE4,0xFF,

This is then repeated with (if I remember correctly) 10ms delays. I did look at the message and tried to see if I could figure something out. At first I thought that it was a MODE B, but with more channels, but when I applied this, the CRC calculation failed. That is when I tested the previous (i.e. MODE B) method on the "inner" message instead. The CRC calculation matches and the result can be converted to channel data in the same way as for standard MODE B. So, for this implementation I simply "discard" the outer part of the message (but it does sync on the first 0xA1 byte. It seems to work very good, but I am not very happy with not using all of the bytes received. Note also that the baud rate is not the same as for the standard XBUS receiver.

Maybe you can see any pattern in the data not used? I do not think it is an empty frame as it "surrounds" the MODE B frame I am using. However all ideas on interpreting this are welcome!